### DISCUSSION TITLE
'Call For Comments' Discussion Category

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#1093] [Announcements]
Community, in order to refine the F´ process a bit more, we have created the "Call For Comments" discussion category.  This category will be used to solicit comments from you all on improvements that the F´ team intends to make.  This will help us ensure we have reflected upon community opinions before making substantial changes to F´.

Please continue discussing ideas and suggestions for F´  in the normal discussion topics you have been using.  However, when the F´ team is ready to tackle  a given feature, change, or improvement we'll post a formal "Call for Comments" ticket where you can weigh-in on the proposal from the team.  These will be in the category "Call for Comments".

Thanks!

**Comment by ThibFrgsGmz** [comment]
Since April 12, GitHub has officially added the [Polls feature](https://github.blog/changelog/2022-04-12-discussions-polls/) to Discussions. Maybe it would be very interesting to use them for "Call For Comments"? This is to graphically visualize the sensitivity of the community towards a considered feature.

Official presentation video by the GitHub team: https://www.youtube.com/watch?v=EWckNPIhXuQ
